test/hotspot/jtreg/runtime/CommandLine/VMDeprecatedOptions.java

```diff
@@ -62,6 +62,14 @@ public class VMDeprecatedOptions {
             {"CreateMinidumpOnCrash", "false"}
           }
         ));
+        if (Platform.isX86() || Platform.isX64()) {
+          deprecated.addAll(
+            Arrays.asList(new String[][] {
+              {"UseRTMLocking",         "false"},
+              {"UseRTMDeopt",           "false"},
```

Comment:
Should `RTMRetryCount` be mentioned here as well?

Reply:
Done. I thought I can only use boolean flags there.